### PR TITLE
bug: when application starts homepage lands on localhost:3000/Fornear

### PR DESCRIPTION
**Pull request by neeravbhaskarla**
Homepage lands on localhost:3000/Fornear, on `npm start`
## Description
Changed Homepage in `package.json` from `https://jibranabsarulislam.github.io/Fornear` to `https://jibranabsarulislam.github.io/`

## Related Issue

Issue: #50 

## Screenshots (if appropriate):

https://github.com/It-s-Saturday/Fornear/assets/57148990/b39b53fe-4f9e-418c-a2c8-c95272c7cdf6

## Types of changes

- [x] Bug fix
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation
- [ ] Other (please describe)

### CODE REVIEW

**Comment by jibranabsarulislam** [comment]
thanks again @neeravbhaskarla 

i had someone else on this issue to learn the process, but it's been a couple days so we'll go with your PR

File: frontend/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://jibranabsarulislam.github.io/Fornear/",
+  "homepage": "https://jibranabsarulislam.github.io/",
```
Comment (jibranabsarulislam) [inline frontend/package.json:5]: this works. i will say that this homepage key came from  trying to use github pages during the hackathon, so we don't need it at all anymore. the entire line can be deleted

Comment (neeravbhaskarla) [reply frontend/package.json:5]: okay will remove

File: frontend/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://jibranabsarulislam.github.io/Fornear/",
+  "homepage": "/",
```
Comment (jibranabsarulislam) [inline frontend/package.json:5]:
```suggestion
```

**Review by jibranabsarulislam — CHANGES_REQUESTED** [review]

**Review by jibranabsarulislam — APPROVED** [review]
great, thanks again @neeravbhaskarla and thank you for the video

**Pull request merged**